.gitignore

```diff
@@ -9,4 +9,5 @@ target/
 # Ignore editor config
 .idea
 
-.npmrc
\ No newline at end of file
+.npmrc
+project/emulator-start.sh
```

Comment:
Maybe here in this PR you can include the file for emulator-start.sh.

Reply:
Ack, I'll put it in a new PR so it doesn't continue to muddle up this PR's focus